Automattic/jetpack
# Social Logos: update icon font

Fixes #7148

## Proposed changes:

- It adds some new logos, that one can see directly by loading `yoursite.com/wp-content/plugins/jetpack/_inc/social-logos/social-logos.html`
- It removes the handling of an `eot` file when declaring the icon font. This is no longer necessary with modern browsers.

See discussion in #28694 and commit 84c2ecee17ef6d313c296959d0f21c06ead04950 for more details.

This is needed for #28694.


### Other information:


## Jetpack product discussion

* Old discussion: p1HpG7-3An-p2

## Does this pull request change what data or activity we track or use?

* No

## Testing instructions:

* Load `yoursite.com/wp-content/plugins/jetpack/_inc/social-logos/social-logos.html`
* Check the Mastodon logo, it should be displayed properly.
* Go to Jetpack > Settings > Sharing
* Enable the Sharing buttons
* Go to Settings > Sharing
* Play with the settings, the buttons should be displayed properly.
* Go to your site's frontend, the sharing buttons should also be displayed nicely.
* Try in different browsers just to be sure.
